pkg/apis/aerospike/v1alpha2/cluster.go

```diff
@@ -19,6 +19,7 @@ package v1alpha2
 import (
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/api/core/v1"
```

Comment:
The idea here is to use aliases that tell us faster what we're using through the code.

```suggestion
	corev1 "k8s.io/api/core/v1"
```